assets_src/assets/js/ContentElements/QueryBuilderTokenizer.js

```diff
@@ -131,7 +131,7 @@
       /* ADD VALUES */
       $.each(
         vals, function (idx, val) {
-          if(val !== null)
+          if(val !== null && (val !== 0 && val !== ""))
```

Comment:
could these not be valid values?

Reply:
For populating the inputs I don't think an empty string is a valid value, but the 0 is a questionable one. I'll look into an alternative solution to this